client/python/ovmsclient/samples/README.md

```diff
@@ -342,35 +348,37 @@ docker run -d --rm -v ${PWD}/models/public/resnet-50-tf:/models/public/resnet-50
 - Command
 
 ```bash
-usage: http_predict_binary_resnet.py [-h] [--images_dir IMAGES_DIR]
-                              [--service_url SERVICE_URL]
-                              [--model_name MODEL_NAME]
-                              [--model_version MODEL_VERSION]
-                              [--timeout TIMEOUT]
+python http_predict_binary_resnet.py --help
+usage: http_predict_binary_resnet.py [-h] --images_dir IMAGES_DIR
+                                     [--service_url SERVICE_URL]
+                                     [--model_name MODEL_NAME]
+                                     [--model_version MODEL_VERSION]
+                                     [--timeout TIMEOUT]
+
 
 
 optional arguments:
   -h, --help            show this help message and exit 
   --images_dir IMAGES_DIR Path to a directory with images in JPG or PNG format
-  --service_url SERVICE_URL Specify url to http service. default:localhost:9000
+  --service_url SERVICE_URL Specify url to http service. default:localhost:8000
   --model_name MODEL_NAME Model name to query. default: resnet
   --model_version MODEL_VERSION Model version to query. default: latest available
   --timeout TIMEOUT Request timeout. default: 10.0
 ```
 - Usage example
 
 ```
-python http_predict_binary_resnet.py --images_dir images --model_name resnet
-Image images/magnetic_compass.jpeg has been classified as magnetic compass
-Image images/pelican.jpeg has been classified as pelican
-Image images/gorilla.jpeg has been classified as gorilla, Gorilla gorilla
-Image images/snail.jpeg has been classified as snail
-Image images/zebra.jpeg has been classified as zebra
-Image images/arctic-fox.jpeg has been classified as Arctic fox, white fox, Alopex lagopus
-Image images/bee.jpeg has been classified as bee
-Image images/peacock.jpeg has been classified as peacock
-Image images/airliner.jpeg has been classified as warplane, military plane
-Image images/golden_retriever.jpeg has been classified as golden retriever
+python http_predict_binary_resnet.py --images_dir ../../../../demos/common/static/images --model_name resnet
```

Comment:
Why non binary client logs `Image #1 #2 #3 #...` but this client logs image paths?

Reply:
Because binary predict samples take path to directory with images as an output. Therefore it seems good to bind inference results with particular files being inferred.

Non binary predict samples take a single NPY file that contains all data to be inferred. There's no good way to name them, so we just iterate over the numpy array read from the output and output iterator + inference results.